src/components/Authorship.svelte

```diff
@@ -27,6 +28,9 @@
   .body {
     white-space: nowrap;
   }
+  .alias {
+	  color: gray;
```

Comment:
```suggestion
  var(--color-foreground-6)
```

Reply:
that's not the same color

Reply:
`5` works

Reply:
https://github.com/radicle-dev/radicle-interface/compare/647bf9c9d5eee91f89bd059d70aed86eb042f196..78a0d79c49f9110fb40997f0fbbca7040ddb5780

Reply:
Then pick one that matches from our color system: https://github.com/radicle-dev/radicle-interface/blob/master/public/colors.css

Reply:
You can see all available colors in dev mode by pressing <kbd>d</kbd> in the app.

<img width="1840" alt="Screenshot 2023-05-12 at 08 17 40" src="https://github.com/radicle-dev/radicle-interface/assets/158411/2e997e4e-c68c-491b-8be0-06926d49d642">